CHANGES.rst

```diff
@@ -4,8 +4,14 @@ Changelog of controlnext
 
 0.20 (unreleased)
 -----------------
-
-- Nothing changed yet.
+- Added SSO
+- Changed UserProfile model relation to grower to many to many
+- Updated model and views to allow for a many to one relation between grower and
+  basin
+- Added logout buttons
+- Added login selector
+- Only link to url_slug when authorized
+- Added 403 when unauthorized instead of redirect to referer
 
```

Comment:
Punt a/h eind v/d zinnen :-)

Bij dit soort lijstjes zet ik zelf altijd een lege regel tussen de items. Het *kan* zonder, maar voor uitlijnen enzo is het makkelijker. Maar de rest van deze file is op dezefde manier gedaan, dus laat het maar zo voor consistentie.